modules/globebrowsing/src/rawtiledatareader.cpp

```diff
@@ -235,14 +234,14 @@ RawTile::ReadError postProcessErrorCheck(const RawTile& rawTile,
     if (hasMissingData && onHighLevel) {
         return RawTile::ReadError::Fatal;
     }
-    return RawTile::ReadError::None;
+    else {
```

Comment:
Why add the else here?

Reply:
Good question.  This function looked more like a "Choose this or that" if statement.   The way it was written before it read more of a `return RawTile::ReadError::Fatal` is the early out of the function and the `RawTile::ReadError::None` is the "normal control flow" of it.